### PR TITLE
chore: code clean-unused imports and comments

### PR DESCRIPTION
**Pull request by amyXia1994**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [X] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):

**unused imports:**
- kclvm/sema/src/resolver/global.rs
- kclvm/tools/src/LSP/src/main.rs

**unused comments:**
- kclvm/tools/src/LSP/src/find_refs.rs
- kclvm/tools/src/LSP/src/notification.rs

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

This PR clean unused imports and comments.

image
image

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [X] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [X] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [X] Other

### CODE REVIEW

File: kclvm/sema/src/resolver/global.rs [hunk 1]
```diff
@@ -9,7 +9,6 @@ use crate::ty::{
 };
 use indexmap::IndexMap;
 use kclvm_ast::ast;
-use kclvm_ast::walker::MutSelfTypedResultWalker;
```
Comment (Peefy) [inline kclvm/sema/src/resolver/global.rs:12]: Pls add more PR information and tags

Comment (amyXia1994) [reply kclvm/sema/src/resolver/global.rs:12]: added, ptal

Comment (Peefy) [inline kclvm/sema/src/resolver/global.rs:12]: image
Linux CI failed.

Comment (amyXia1994) [reply kclvm/sema/src/resolver/global.rs:12]: rerun and passed

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**